### PR TITLE
DOC: Fix broken link to NEP50.

### PR DESCRIPTION
**Pull request by anushatomar13**
@mattip 
Please let me know if this PR looks fine now? As from what you are saying, I have restored the line break and the next line now starts from a new line.

### CODE REVIEW

**Comment by anushatomar13** [comment]
@mattip , @charris any update on reviewing the PR?

**Comment by anushatomar13** [comment]
@charris Could you please confirm if putting "[skip actions][skip azp][skip cirrus]" in the commit message would skip the build and solve the issue? Because I have only changed the documentation.

**Comment by charris** [comment]
Circle CI is the documentation build, and it is failing.

File: doc/source/release/2.0.0-notes.rst [hunk 1]
```diff
@@ -76,8 +76,8 @@ Highlights of this release include:
 
 - Improved behavior:
 
-  - Improvements to type promotion behavior was changed by adopting `NEP
-    50 <NEP50>`_. This fixes many user surprises about promotions which
+  - Improvements to type promotion behavior was changed by adopting :ref:`NEP 50 <NEP50>`. 
+   | This fixes many user surprises about promotions which
```
Comment (mattip) [inline doc/source/release/2.0.0-notes.rst:80]: Why did you add a vertical bar here?

```suggestion
   | This fixes many user surprises about promotions which
```

**Comment by charris** [comment]
Fixed by #26527.
